README.md

```diff
@@ -12,6 +12,8 @@ If a referenced configuration is changed, **all the pods that use it must be re-
 
 > **_NOTE:_**  The CNI plugin doesn't currently support scenarios that involve IPAM assigning an IP on the pod interface. 
 
+This project provides a subset of features that are otherwise available through [k8snetworkplumbingwg/multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy). However, unlike Multi Network Policies, the bridge-filtering CNI is operating on netfilter's `bridge` table and does not require IPAM, which makes it suitable for KubeVirt.
```

Comment:
> and does not require IPAM

I'm not sure if this plays well with the preceeding note that states that we don't support IPAM.
Wouldn't it be better to leave IPAM out of this and just focus on the fact that we need to filter on bridge table for Kubevirt VMs with bridge bindings?

Reply:
Sounds good, let me fix it